lib/online_order.rb

```diff
@@ -0,0 +1,89 @@
+require 'csv'
+require_relative '../lib/order'
+
+module Grocery
+
+  class OnlineOrder < Order
+
+    attr_reader :id, :products, :status, :customer_obj, :total, :add_product, :find_by_customer
+
+    def initialize (id, products, customer_obj, status)
+      fufillment_status = [:pending, :paid, :processing, :shipped, :complete]
+      @id = id.to_i
+      @products = products
```

Comment:
Instead of saving `@id` and `@products` yourself here, you should call `super(id, products)` and let `Order` do this work for you.

Reply:
Went back and included this. Useful to know, I implemented super at one point and could have utilized it more frequently.